src/FTPClient.jl

```diff
@@ -16,6 +16,7 @@ type FTPClientError <: Exception
        lib_curl_error::UInt32
 end
 
+@enum FTP_MODE ascii_mode binary_mode
```

Comment:
I noticed that there's also an `ascii` in base, and it doesn't get overwritten with `using FTPClient`.
```
julia> workspace()

julia> ascii
ascii (generic function with 3 methods)

julia> using FTPClient
WARNING: using FTPClient.ascii in module Main conflicts with an existing identifier.

julia> ascii
ascii (generic function with 3 methods)

julia> binary
binary
```
I don't think there's a simple way to have both asciis coexists, so just keep it as `ascii_mode` and `binary_mode`?